src/main/java/com/billingapi/controller/InvoiceController.java

```diff
@@ -13,39 +15,41 @@
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 
+
 import javax.validation.Valid;
 
+
 @RestController
 @RequestMapping("/invoice")
 @Validated
 public class InvoiceController {
-    static Logger logger= LogManager.getLogger(InvoiceController.class);
+    static Logger logger = LogManager.getLogger(InvoiceController.class);
```

Comment:
Can we use @Slf4j. and please explore more about this and implement in a proper way.